spec/predicates/vsa.md

```diff
@@ -1,11 +1,358 @@
-# Predicate type: SLSA Verification Summary (VSA)
+# Predicate type: Verification Summary
 
-Type URI: https://slsa.dev/verification_summary/v1
+Type URI: https://in-toto.io/attestation/verification_summary/v1
 
-Version: 1.0
+Version: 1.1
 
 ## Purpose
 
-Records a SLSA verification decision about an artifact.
+Verification summary attestations communicate that an artifact has been verified
+against some policy and details about that verification. Such details may
+include, but are not limited to, what [SLSA](https://slsa.dev) level the
+artifact and/or its dependencies were verified to meet.
 
-For more details, please see https://slsa.dev/spec/v1.0/verification_summary
+## Use Cases
+
+Software consumers make a decision about the validity of an
+artifact without needing to have access to all of the attestations about the
+artifact or all of its transitive dependencies.  They delegate
+complex policy decisions to some trusted party and then simply trust that
+party's decision regarding the artifact. That decision is represented by a VSA.
+
+VSAs also allow software producers to keep the details of their build pipeline
+confidential while still communicating that some verification has taken place.
+This might be necessary for legal reasons (keeping a software supplier
+confidential) or for security reasons (not revealing that an embargoed patch has
+been included).
+
```

Comment:
Another use case is that verification sometimes can be costly and not suitable for certain scenarios. 

Something like [Enterprise Contract](https://enterprisecontract.dev/) can evaluate policies based on an image's labels or even one of its file's contents. This wouldn't really be possible to perform as a Kubernetes admission web hook.